### PR TITLE
[Sample] Add AI platform sample test

### PR DESCRIPTION
**Pull request by numerology**
Part of #1813

Memo: Currently using a ad hoc logic in presubmit-tests-with-pipeline-deployment.sh to GC the generated models and versions. Long term solution should be that the notebook cleans up the generated files itself, but it requires further investigation on how to grant proper permission.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2236)

### CODE REVIEW

**Comment by Ark-kun** [comment]
What is the benefit of renaming the samples and files?

**Comment by numerology** [comment]
> What is the benefit of renaming the samples and files?

Currently, the sample test infra assumes that the test file and the dir have the same name. 
But I'll make a better name after making this work.

**Comment by Ark-kun** [comment]
> Currently, the sample test infra assumes that the test file and the dir have the same name.

Would you mind if I fix that code at some point? (I won't block your work)

**Comment by numerology** [comment]
> > Currently, the sample test infra assumes that the test file and the dir have the same name.
> 
> Would you mind if I fix that code at some point? (I won't block your work)

Sure, that will be appreciated

**Comment by numerology** [comment]
I suspect there's some ongoing issue about using GOOGLE_APPLICATION_CREDENTIALS within notebook, need further investigation.

See [here](https://github.com/kubeflow/kubeflow/issues/4190)

**Comment by numerology** [comment]
TODO: switch back to create_run_from_pipeline_func once it works in test infra.

File: samples/core/ai-platform/ai-platform.ipynb [hunk 1]
```diff
@@ -257,16 +275,42 @@
...
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "kfp.Client().create_run_from_pipeline_func(pipeline, arguments={}, experiment_name=EXPERIMENT_NAME)"
+    "# Specify pipeline argument values\n",
+    "arguments = {}\n",
+    "\n",
+    "# Submit a pipeline run\n",
+    "run_name = pipeline_func.__name__ + ' run'\n",
```
Comment (Ark-kun) [inline samples/core/ai-platform/ai-platform.ipynb]: Is there any reason to set the run name?

Comment (numerology) [reply samples/core/ai-platform/ai-platform.ipynb]: Actually no. Removed.

**Comment by gaoning777** [comment]
Looks like the service accounts are working fine. PTAL: https://github.com/kubeflow/pipelines/pull/2327

**Comment by numerology** [comment]
/close
#2327  is a better solution.

**Pull request closed without merging**